alerting/src/main/kotlin/org/opensearch/alerting/model/Monitor.kt

```diff
@@ -77,6 +79,22 @@ data class Monitor(
                     require(trigger is DocumentLevelTrigger) { "Incompatible trigger [${trigger.id}] for monitor type [$monitorType]" }
             }
         }
+
+        // validate the number of Actions Across Triggers
+        if (MonitorRunnerService.monitorCtx.totalMaxActionsAcrossTriggers > AlertingSettings.UNBOUNDED_ACTIONS_ACROSS_TRIGGERS) {
+            val totalActions = triggers.sumOf { trigger ->
+                require(trigger.actions.size <= MonitorRunnerService.monitorCtx.maxActionsAcrossTriggers) {
+                    "For a single trigger, the number of actions to be executed must be less than or equal to " +
+                        "${MonitorRunnerService.monitorCtx.maxActionsAcrossTriggers}"
+                }
+                trigger.actions.size
+            }
+            require(totalActions <= MonitorRunnerService.monitorCtx.totalMaxActionsAcrossTriggers) {
```

Comment:
For this, dont we want to validate the total number of actions across monitors and triggers? So based on that, we need to do a search call to see the total number of actions across triggers and monitors